### PR TITLE
update splash screen and animations tutorials for Godot 3.0

### PR DESCRIPTION
**Pull request by brainsick**
#946

### CODE REVIEW

**Comment by brainsick** [comment]
I was thinking that I might follow up with another PR for further word smithing, an updated animation gif/result, and any changes needed because I didn't actually build the HTML and preview the results.  Scandalous, I know!

**Comment by brainsick** [comment]
image
image

**Comment by mhilbrunner** [comment]
Looks good. @cbscribe should take a look, as he knows the step-by-step section way better than me :)

File: learning/step_by_step/animations.rst [hunk 1]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
 just in case: :download:`robisplash.zip <files/robisplash.zip>`.
 
-Creating the animation
-----------------------
+Add an animation player
+-----------------------
 
 First of all, add an :ref:`AnimationPlayer <class_AnimationPlayer>`
-node to the scene as a child of "bg" (the root node):
+node to the scene as a child of "background" (the root node):
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: We should try and consistently follow the style guide conventions in all docs: http://docs.godotengine.org/en/latest/learning/scripting/gdscript/gdscript_styleguide.html

Which means the node should be named "Background" (capital "B").

File: learning/step_by_step/animations.rst [hunk 2]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
+It's time to create a new animation! Press the new animation button and name
+the animation "intro" when the dialog appears.
 
-.. image:: img/animeditor.png
+.. image:: img/robisplash_anim_new.png
 
-The second and most important, is that the property editor enters
+Now that we have an animation the property editor enters
 "animation editing" mode. In this mode, a key icon appears next to
 every property of the property editor. This means that, in Godot, *any
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: too many commas. In Godot any property of an object...

File: learning/step_by_step/animations.rst [hunk 3]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
-this, select the "logo" node and go to the "pos" property, move it up
-to position: 114,-400.
+The logo will appear from the top of the screen.
 
-Once in this position, press the key button next to the property:
+With the animation editor panel open, select the "logo" node and set the "Rect / Position"
+property to: 118, -400.
 
-.. image:: img/keypress.png
+Once in this position, press the key button next to the property:
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: I'd just combine these two sentences:

property to ``(110, -400)`` and press the key button next to the property.

File: learning/step_by_step/animations.rst [hunk 4]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
-.. image:: img/keyadded.png
+.. image:: img/robisplash_anim_editor_keyframe.png
 
 Move the editor cursor to the end, by clicking here:
 
-.. image:: img/move_cursor.png
+.. image:: img/robisplash_anim_editor_track_cursor.png
 
-Change the logo position to 114,0 and add a keyframe again. With two
+Change the logo position to 118, 0 and add a keyframe again. With two
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: In general, most docs highlight variables and values with code tags to make them more visible: ``(118, 0)``

File: learning/step_by_step/animations.rst [hunk 5]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
 keyframes, the animation happens.
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: This sentence is bad (I know it's not yours). Plus, nothing actually happens yet, so it's misleading.

File: learning/step_by_step/animations.rst [hunk 6]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
 
-.. image:: img/animation.png
+.. image:: img/robisplash_anim_editor_keyframe_2.png
+
+Pressing "play from start" on the animation panel will make the logo descend.
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: The actual text of the popup is "Play selected animation from start. (Shift-D)"

File: learning/step_by_step/animations.rst [hunk 7]
```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
...
+
+.. image:: img/robisplash_anim_editor_play_start.png
 
-Pressing Play on the animation panel will make the logo descend. To test
-it by running the scene, the autoplay button can tag the animation to
+To test it by running the scene, the autoplay button can tag the animation to
```
Comment (cbscribe) [inline learning/step_by_step/animations.rst]: autoplay isn't "for" testing. Try

Click the "Autoplay on Load" button to set the animation to start automatically when the scene starts.

**Comment by brainsick** [comment]
I made the requested text changes despite word smithing being outside of my intended scope.

I don't plan to update the sample project and screenshots again though.  I can close the PR if that's preferred.

**Comment by mhilbrunner** [comment]
I'll merge this. Better than leaving it as-is.

Thanks for your work!

**Pull request merged**